client.go

```diff
@@ -68,11 +68,10 @@ func (c Client) Send(filename string, mode string) (io.ReaderFrom, error) {
 		s.opts["blksize"] = strconv.Itoa(c.blksize)
 	}
 	n := packRQ(s.send, opWRQ, filename, mode, s.opts)
-	addr, err := s.sendWithRetry(n)
+	err = s.sendWithRetry(n)
 	if err != nil {
 		return nil, err
 	}
-	s.addr = addr
```

Comment:
Sounds great!

I'm using `go test -v` in `tftp` directory to run all the tests.

And i.e. `go test -v -run=TestNotFound` in case there is some single stubborn test that fails.